SU-SWS/stanford_profile
# D8CORE-1452: update line lengths

# READY FOR REVIEW
- (Edit the above to reflect status)

# Summary
- This adjusts the line lengths.

# Needed By (Date)
- Soon

# Urgency
- not terribly.

# Steps to Test

1. Check out this branch
2. Clear caches
3. Verify that the headings have the expected `max-width` values:
H2:  max-width: 40ch;
H3:  max-width: 48ch;
H4: max-width: 60ch;
H5: max-width: 75ch;
Drop Cap: max-width: 55ch;
Callout: max-width: 48ch;
kill: width: 80%

# Affected Projects or Products
- D8Core

# Associated Issues and/or People
- JIRA ticket: https://stanfordits.atlassian.net/browse/D8CORE-1452
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)
@kerri-augenstein 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
